weave/trace/settings.py

```diff
@@ -73,6 +73,30 @@ class UserSettings(BaseModel):
     This cannot be changed after the client has been initialized.
     """
 
+    use_server_cache: bool = False
+    """
+    Toggles caching of server responses, defaults to False
+
+    If True, caches server responses to disk.
+    Can be overridden with the environment variable `WEAVE_USE_SERVER_CACHE`
+    """
+
+    server_cache_size_limit: int = 1_000_000_000
+    """
+    Sets the size limit in bytes for the server cache, defaults to 1GB (1_000_000_000 bytes).
+    Required if `use_server_cache` is True.
```

Comment:
It will always be specified.  Maybe rephrase as:
```
Ignored if `use_server_cache` is `False`
```

Reply:
changed